docs/user_docs/cli/kbcli_login.md

```diff
@@ -0,0 +1,47 @@
+---
+title: kbcli login
+---
+
+Login command.
+
+```
+kbcli login [flags]
+```
+
+### Options
+
+```
+  -c, --context string   Context name.
+  -h, --help             help for login
```

Comment:
help -> Help

Reply:
Sorry, `-h` is a built-in option of `cobra.Command`, and the description information of `help for login` is automatically generated.